API.md

```diff
@@ -0,0 +1,124 @@
+# API Documentation
+
+- Path: `<host>/api/v2`
+
+## Directives
+
+### POST `/directive`
+
+- Headers: Content-Type: application/json, Authorization: Bearer \<JWT from FusionAuth>
+- Request Body
+
+```
+{
+  archiveId: string,
+  stewardEmail: string (email format),
+  type: string (must be "transfer" presently),
+  note: string (optional),
+  trigger: {
+    type: string (must be "admin" presently)
+  }
+}
+```
+
+- Response
+
+```
+{
+  directiveId: string,
+  archiveId: string,
+  type: string,
+  createdDt: string (date),
+  updatedDt: string (date),
+  trigger: {
+    directiveTriggerId: string,
+    directiveId: string,
+    type: string,
+    createdDt: string (date),
+    updatedDt: string (date),
+  },
+  stewardAccountId: string,
+  note: string,
+  executionDt: string (date)
+}
+```
+
+### POST `/directive/trigger/account/:accountId`
+
```

Comment:
This one is a little less intuitive so it might be helpful to add a sentence here like "Execute an existing directive"

Reply:
(Thinking more for mobile team than Natalie in this case)